### ISSUE TITLE
[Request] iPad Orientation

### ISSUE THREAD

**Issue by korboybeats** [s8ngyu/Mugunghwa#19]
May you please implement iPad orientation? Thank you

**Comment by s8ngyu** [comment]
Doesn't it work on iPads? I've disabled landscape orientation due to possible UI mess up, I'll test this and update!

**Comment by korboybeats** [comment]
> Doesn't it work on iPads? I've disabled landscape orientation due to possible UI mess up, I'll test this and update!

Yeah portrait mode works, but I would love to see a landscape orientation if possible. Thank you
